scripts/gcloud-shutdown.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
```

Comment:
please add as helper function to `gcloud.sh` akin to the other delete helper functions.

Reply:
This is added as metadata for the VM, so it needs to be isolated. As an alternative, I could generate this file during the CD workflow instead of having it in our repository as a file.